.CI/cmake/Jenkinsfile.cmake.ubuntu.gcc.test

```diff
@@ -25,7 +25,7 @@ pipeline {
         stage('autoconf-jammy-gcc') {
           agent {
             docker {
-              image 'docker.openmodelica.org/build-deps:v1.22.2'
+              image 'anheuermann/openmodelica-build-deps:v1.22.1-qtwebengine'
```

Comment:
This should not be changed yet. It is not relevant for this yet. It is used for the nightly builds on master.

Your CMake builds are now working on Jenkins. So the issues left are with the autoconf build and, of course, the logic and adjustments related to the original intention of the PR.

Reply:
ok